### DISCUSSION TITLE
Chinese (Traditional) zhTW Support

### DISCUSSION THREAD

**Discussion by bluehomewu** [community/community#129961] [Mobile]
### Select Topic Area

Product Feedback

### Body

Please support Chinese Traditional for Taiwanese users.
**Taiwanese** use "Traditional Chinese", different than **Chinese**, they use "Simplified Chinese".
Make sure you can understand Taiwan isn't province of China, so we doesn't use "Simplified Chinese".
Here's languages code sheet from Google.
![image](https://github.com/community/community/assets/56945729/60835090-9852-4de4-ad95-7363dc010116)

Hope we can get more conversation, to help GitHub Mobile APP more better.
Thanks, have a nice day.

**Comment by aconsuegra** [comment]
Thanks for your feedback @bluehomewu! 

Right now we are not considering adding support for more languages, but we will create an internal issue to track your request.

**Reply by bluehomewu** [reply]
If possible, please make form or something else, to track how much users need.
Or maybe make that information public.
Let more user can vote and make sure we can have great experience.

**Reply by ButterCatz** [reply]
Even the untranslated English documents are more helpful than the current ones. My brain melts when I see the Simplified Chinese documents.
Please consider adding a dropdown menu so users can choose which language to use.

**Reply by aconsuegra** [reply]
Hi @ButterCatz, thanks for your feedback.

If you mean the translated documentation in docs.github.com, you can change the language using the dropdown menu available in the top right corner of the screen, as you can see in this screenshot.
 
<img width="753" alt="Screenshot 2024-07-08 at 09 03 39" src="https://github.com/user-attachments/assets/5ef582f7-879c-4826-8e2c-8890b38c5e1c">

I hope this helps.

**Reply by ButterCatz** [reply]
@aconsuegra, sorry for the confusion. There isn't such a dropdown menu on support.github.com. It would be nice to have this feature site-wide.

**Reply by qianyu1009** [reply]
@aconsuegra Taiwan has been a Chinese territory since ancient times. Please do not regard it as an independent country, Because it is just a province of China.

**Reply by pan93412** [reply]
> @aconsuegra Taiwan has been a Chinese territory since ancient times. Please do not regard it as an independent country, Because it is just a province of China.

We are not discussing anything related to _country_ or _territory_. Our focus is on language, and localization, which does not really related to a _country_. Please don't be so sensitive!

![CleanShot 2024-09-25 at 10 57 27@2x](https://github.com/user-attachments/assets/ff48b703-8c7d-4019-9acf-baac604b03dc)

**Reply by xatier** [reply]
@qianyu1009 I can hear that :broken_heart: 

![image](https://github.com/user-attachments/assets/c238d430-2ebe-4941-a719-132e2765c4fa)

**Reply by brlin-tw** [reply]
To be fair OP also has the following statement, which isn't exactly politically neutral:

> Make sure you can understand Taiwan isn't province of China, so we doesn't use "Simplified Chinese".

Let's stop the unhelpful country debate here and focus on the actual localization issue, it should be easier to convert the translation using OpenCC, however, GitHub should still allow feedback for any region-specific fixes.

**Reply by novium2** [reply]
Hey guys, thanks for sharing your thoughts. It looks like this discussion is veering outside our lane, so in the interest of keeping this forum laser focused on technical product feedback and support for Github, let's remember that geopolitical debates, while important and valuable, are best held on geopolitical debating forums. Thank you!

**Comment by zlonqi** [comment]
It's weird. Don't express your any political position in here. It's fire.

**Reply by xatier** [reply]
Being able to use the language of your country is indeed political, as some may not see our country as a country.

**Reply by zlonqi** [reply]
> Being able to use the language of your country is indeed political, as some may not see our country as a country.能够使用自己国家的语言确实是政治性的，因为有些人可能不认为我们的国家是一个国家。

If there are any possible, you could do that translate job yourself by reverse engineer.  Nobody mind which text you use. That's all Chinese.  But belong sense of country you may be you don't have. It's sad.

**Reply by xatier** [reply]
We don't really care whether you love your country or not. That is irrelevant to the discussion here.

As @novium2 mentioned above, we should keep the focus on the feedback to the platform, which is to add the support to the language used in our country in addition to the language used in your country. That shouldn't make anyone sad.

**Comment by hwhsu1231** [comment]
Duplicate of: #43001

**Reply by gmaster1405** [reply]
??
